audit/projects/k8s-infra-e2e-boskos-001/buckets/kubernetes-staging-485128143e-asia/bucketpolicyonly.txt

```diff
@@ -0,0 +1,3 @@
+Bucket Policy Only setting for gs://kubernetes-staging-485128143e-asia:
```

Comment:
This is why test SAs shouldn't be able to make new buckets, I guess?  How do we root cause it?

Reply:
Covered this in https://github.com/kubernetes/k8s.io/issues/1664#issuecomment-785970376

I'll open a followup issue for how to avoid these growing unbounded